Dockerfile.develop

```diff
@@ -2,10 +2,8 @@ FROM node:18-alpine AS base
 WORKDIR /opt/isomercms-backend
 RUN mkdir /root/.ssh
 COPY . .
-COPY ./.ssh /root/.ssh
-RUN chmod 600 /root/.ssh/github.pub
-RUN chmod 600 /root/.ssh/github
 RUN apk update
+RUN apk add --no-cache bash
```

Comment:
nit#1: we can consider combing all these `RUN` commands in a single command:
```
RUN apk update && \
    apk add --no-cache bash git openssh-client
```
nit#2: add a line to remove apk cache after the installation

Reply:
sure, sounds god